Gemfile.lock

```diff
@@ -265,6 +266,7 @@ GEM
     thor (0.20.3)
     thread_safe (0.3.6)
     timecop (0.9.1)
+    truncate_html (0.9.3)
```

Comment:
This gem seems a bit outdated and with some [potential issues](https://github.com/hgmnz/truncate_html/issues) that could affect the course output (e.g. the Unicode and emojis issues) – I notice it uses regular expressions, which makes it quite error prone (I think). Would it be worth switching to something like https://github.com/jorgemanrubia/truncato, which uses Nokogiri under the hood + is more recent and up to date?

Reply:
Updated :)